wall_input_topic: "laser_wall_extraction/viz_req"
#clustering_input_topic: "laser_clustering/clusters"
clustering_input_topic: "laser_overlap_trace/clusters"
overlap_input_topic: "laser_overlap_trace/tracks"
num_of_diagrams: 2
pause_function: False
